### PR TITLE
Fix LinkedIn metatags

### PR DESCRIPTION
**Pull request by adrianoapmartins**

### CODE REVIEW

**Comment by adrianoapmartins** [comment]
LinkedIn preview is working: https://www.linkedin.com/post-inspector/inspect/https:%2F%2Fwebsite-git-fix-linkedin-metatags-reviewpad.vercel.app%2Fdevtoolsporto

File: components/Page.tsx [hunk 1]
```diff
@@ -26,21 +28,23 @@ export default function Page({ title, description, url, imgUrl, children }: Prop
...
-        url={url ? url : EnvVars.URL}
         title={title} 
-        date={""} 
+        slug={slug}
+        // date={""} 
         description={description} 
-        tags={""} 
-        author={"Reviewpad"} 
+        // tags={""} 
+        // author={"Reviewpad"}
```
Comment (ferreiratiago) [inline components/Page.tsx:36]: Please delete commented lines

**Review by ferreiratiago — APPROVED** [review]

**Comment by adrianoapmartins** [comment]
I don't think https://metatags.io/ is generating tags correctly at all. The current version is tested and working on:

- [LinkedIn](https://www.linkedin.com/post-inspector/inspect/https:%2F%2Fwebsite-git-fix-linkedin-metatags-reviewpad.vercel.app%2Fabout-us)
- [Facebook](https://developers.facebook.com/tools/debug/?q=https%3A%2F%2Fwebsite-git-fix-linkedin-metatags-reviewpad.vercel.app%2Fabout-us)
- [Twitter](https://twitter.com/intent/tweet?url=https:%2F%2Fwebsite-git-fix-linkedin-metatags-reviewpad.vercel.app%2Fabout-us)
- [Slack](https://user-images.githubusercontent.com/38539/198883660-24622e1e-a23c-4264-b317-ee15ca99cfa0.png)
- [Discord](https://user-images.githubusercontent.com/38539/198883748-23cb6f20-b8c6-4796-b3b2-1c53b3afbcc5.png)
- [Socialsharepreview](https://socialsharepreview.com/?url=https://website-git-fix-linkedin-metatags-reviewpad.vercel.app/about-us)
- [Linkpreview.dev](https://linkpreview.dev/?url=https%3A%2F%2Fwebsite-git-fix-linkedin-metatags-reviewpad.vercel.app%2Fabout-us%20)

**Comment by adrianoapmartins** [comment]
The changes worked perfectly on this PR's deployment preview. After merging the PR it is working on the alternative vercel domain (https://website-kappa-hazel.vercel.app/), but it is not on the main link, reviewpad.com. Which is weird because it is the exact same deployment.

Meta tag testing for website-kappa-hazel.vercel.app:

- [LinkedIn](https://www.linkedin.com/post-inspector/inspect/https:%2F%2Fwebsite-kappa-hazel.vercel.app%2Fabout-us)
- [Facebook](https://developers.facebook.com/tools/debug/?q=https%3A%2F%2Fwebsite-kappa-hazel.vercel.app%2Fabout-us)
- [Twitter](https://twitter.com/intent/tweet?url=https:%2F%2Fwebsite-kappa-hazel.vercel.app%2Fabout-us)
- [Socialsharepreview](https://socialsharepreview.com/?url=https://website-kappa-hazel.vercel.app/about-us)
- [Linkpreview.dev](https://linkpreview.dev/?url=https%3A%2F%2Fwebsite-kappa-hazel.vercel.app%2Fabout-us%20)

Meta tag testing for reviewpad.com:

- [LinkedIn](https://www.linkedin.com/post-inspector/inspect/https:%2F%2Freviewpad.com%2Fabout-us)
- [Facebook](https://developers.facebook.com/tools/debug/?q=https%3A%2F%2Freviewpad.com%2Fabout-us)
- [Twitter](https://twitter.com/intent/tweet?url=https:%2F%2Freviewpad.com%2Fabout-us)
- [Socialsharepreview](https://socialsharepreview.com/?url=https://reviewpad.com/about-us)
- [Linkpreview.dev](https://linkpreview.dev/?url=https%3A%2F%2Freviewpad.com%2Fabout-us%20)

**Comment by adrianoapmartins** [comment]
The source code is the same in both pages... my only guess is that the domain reviewpad.com must be cached somewhere between vercel and the outside world servers...

**Pull request merged**